inference-engine/tests/functional/plugin/cpu/single_layer_tests/reduce_ops.cpp

```diff
@@ -155,186 +240,424 @@ const std::vector<ngraph::helpers::ReductionType> reductionLogicalTypes = {
         ngraph::helpers::ReductionType::LogicalAnd
 };
 
-const std::vector<std::vector<size_t>> inputShapes = {
-        std::vector<size_t>{10, 5, 15, 12},
-        std::vector<size_t>{3, 5, 7, 9},
+std::vector<std::vector<ov::test::InputShape>> inputShapes = {
+    {{{}, {{2, 19, 2, 9}}}},
+    {{{-1, {2, -1}, -1, -1}, {{2, 9, 2, 9}, {2, 19, 2, 9}, {2, 9, 2, 15}}}},
+    {{{{1, 5}, {2, 5}, {1, 5}, {1, 20}}, {{3, 2, 3, 2}, {3, 3, 3, 2}, {3, 2, 3, 3}}}}
+};
+
+std::vector<std::vector<ov::test::InputShape>> inputShapes_5D = {
+    {{{}, {{2, 19, 2, 2, 9}}}},
+    {{{-1, {2, -1}, -1, -1, -1}, {{2, 9, 2, 2, 9}, {3, 9, 2, 2, 9}, {2, 9, 2, 5, 9}}}},
```

Comment:
Regarding target shapes, is not it better to have some target shapes with more than one block per channel dimension?

Reply:
Fixed. Thanks Maksim!